polkadot/node/core/pvf/common/src/worker/mod.rs

```diff
@@ -60,6 +64,31 @@ macro_rules! decl_worker_main {
 					println!("{}", $worker_version);
 					return
 				},
+
+				"--check-can-enable-landlock" => {
+					#[cfg(target_os = "linux")]
+					let status = if security::landlock::check_is_fully_enabled() { 0 } else { -1 };
+					#[cfg(not(target_os = "linux"))]
+					let status = -1;
+					std::process::exit(status)
+				},
+				"--check-can-unshare-user-namespace-and-change-root" => {
+					#[cfg(target_os = "linux")]
+					let status = if security::unshare_user_namespace_and_change_root(
+						WorkerKind::Execute,
+						&std::env::temp_dir(),
```

Comment:
Here we just check if we *can* do the pivot_root, no files are created here.

Can this still cause a conflict? I'd rather not have any randomness here in the worker, since we want to block e.g. the `getrandom` syscall.

Reply:
you're right, I don't see why calling `pivot_root` would cause any conflicts if we don't create any files 👍🏻